src/tiles/styles.scss

```diff
@@ -11,6 +11,7 @@
 @use '../column-layout/styles.scss' as mixins;
 
 $gutter: awsui.$space-grid-gutter;
+$updated-gutter: calc(#{$gutter} + 4px);
```

Comment:
is this fix an anti-pattern? The `#gutter` variable can always be changed, therefor adding 4 pixels to it will not always help. Doing so here sets the combined margins to be on-par with a 4px convention. (12px, 6px on each side)

Reply:
I think it's a bit confusing, why not create a new design token? other components has their own spacing gutter token https://github.com/cloudscape-design/components/blob/main/style-dictionary/visual-refresh/spacing.ts

Reply:
Thank you, this is the guidance I was searching for.